### DISCUSSION TITLE
[Can someone help me?] Repository home page license info show  [Unknown licenses found]

### DISCUSSION THREAD

**Discussion by casionone** [community/community#32030] [Other Feature Feedback, Questions, & Ideas]
On the home page of our project's repos(https://github.com/apache/incubator-linkis), the license info is displayed as [Apache-2.0, Unknown licenses found]. Before we placed the license-header file in the directory, the home page still displayed after we removed it.
![image](https://user-images.githubusercontent.com/7869972/185275325-a178d3bf-95bd-4d2a-980c-7cf6b87fd9ac.png)

issue https://github.com/apache/incubator-linkis/issues/2701
pr https://github.com/apache/incubator-linkis/pull/2715

**Comment by pawellabaj** [comment]
I have the same situation in https://github.com/pawellabaj/auto-record

**Reply by pawellabaj** [reply]
It's not the case anymore. I did nothing.
Probably the lionk has been "regenerated" after some time

**Comment by pawellabaj** [comment]
@casionone I see that it has been resolved in your repository. What did you do?

**Reply by longquanzheng** [reply]
+1

**Reply by casionone** [reply]
I did nothing, 
Maybe github has fixed it and solved it

**Comment by casionone** [comment]
> @casionone I see that it has been resolved in your repository. What did you do?
